src/components/element-editor/prefill/prefill-child.tsx

```diff
@@ -0,0 +1,88 @@
+import { Form } from "@lifesg/react-design-system/form";
+import { ChildEntry } from "src/components/common";
+import { IPrefill } from "src/context-providers";
+import { FieldWrapper } from "./prefill.styles";
+
+interface IProps {
+    onDelete: () => void;
+    onChange: (newValue: IPrefill) => void;
+    value?: IPrefill;
+}
+
+export const PrefillChild = ({ onDelete, onChange, value }: IProps) => {
+    // =============================================================================
+    // CONST, STATE, REFS
+    // =============================================================================
+    const options = ["myInfo", "Previous source"];
```

Comment:
apologies for missing this, though this differs from the copy from Figma, `Myinfo` should be spelt this way in accordance to their CI guide

```suggestion
    const options = ["Myinfo", "Previous source"];
```